addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -2181,6 +2181,60 @@ QUnit.test('chat window: chat header should not be clickable when thread has mul
     );
 });
 
+QUnit.test('chat window should open when receiving a new DM', async function (assert) {
+    assert.expect(4);
+
+    this.data['mail.channel'].records.push({
+        channel_type: 'chat',
+        id:11,
+        is_pinned: false,
+        members: [this.data.currentPartnerId, 11],
+        uuid: 'channel11uuid',
+    });
+    this.data['res.partner'].records.push({
+        id: 11,
+        name: "lol",
```

Comment:
Please you a proper partner name.

I don't mind funny values, but when these are too far away to something that looks like one, it becomes harder to understand test. E.g. something like "Lolly" is fine (starts with capital letter, looks like a name for a human being, etc.).